testing/src/scenario/_runtime.py

```diff
@@ -345,6 +347,9 @@ def exec(
                 raise UncaughtCharmError(f'Uncaught {type(e).__name__} in charm, try "exceptions [n]" if using pdb on Python 3.13+. Details: {e!r}') from e  # fmt: skip
 
             finally:
+                if ops:
+                    ops._destroy()
```

Comment:
This makes me think that `destroy` should be public (within the private `Ops` and `_Manager` classes).

Reply:
done, thanks!